README.md

```diff
@@ -13,19 +13,46 @@ Implement the tool `st` that does exact pattern matching using a suffix tree. Te
 
 # Report
 
-## Specify if you have used a linear time or quadratic time algorithm.
+## Implementation
+We tried implemented both the naive quadratic time algorithm and McCreight’s linear time algorithm. But when timing our implementation of McCreight’s algorithm, we realised we forgot a crucial edge case, which made it run as slow as the naive. We did not fix it before the deadline. 
```

Comment:
Ok. But now you have a whole week without lectures, so there is plenty of time to fix McCreight :)